README.md

```diff
@@ -12,24 +12,45 @@ The backend of workflow is kept in a separate open-source repo [here](https://gi
 
 The Angular frontend and associated Scala API for Workflow. 
 
-### Install
-
-Development prerequisites
--------------------------
+### Install Prerequisites
 
  * jenv
  * nvm
  
 You will need `workflow` and `capi` (API Gateway invocation) credentials from Janus.
 
-In order to run, workflow-frontend needs to talk to the CODE workflow datastore. It does this via an SSH tunnel to a 
-workflow-frontend CODE instance.
+In order to run, workflow-frontend needs to talk to the a workflow datastore. 
+It can use either a local store, or CODE.
+
+### Common installation steps
+
+ 1. Make sure that you are running the right version of nodejs, (tested and working with v6.1.0). We recommend using [nvm](https://github.com/creationix/nvm) to easily manage multiple versions of node. With this you can use `nvm use` to switch to this version quickly.
+ 1. Run the install script `./scripts/setup.sh`
+ 1. Download the DEV config: `./scripts/fetch-config.sh`
+ 1. Run the `setup-app.rb` in the `dev-nginx` repo with the `nginx/nginx-mapping.yml` file in this repo
```

Comment:
In this numbered list, all the points are marked as 1

Reply:
This is one way you can declare an ordered list, and it will automatically number each item as appropriate (see https://www.markdownguide.org/basic-syntax/#lists)

Reply:
Oh cool, didn't realise that 👍